tf-psa-crypto/tests/suites/test_suite_psa_crypto_storage_format.function

```diff
@@ -293,13 +302,32 @@ void key_storage_read(int lifetime_arg, int type_arg, int bits_arg,
     psa_set_key_algorithm(&attributes, alg);
     psa_set_key_enrollment_algorithm(&attributes, alg2);
 
+    /* Create a persistent key which is intentionally larger than the specified
+     * bit size. */
+    if (flags & TEST_FLAG_OVERSIZED_KEY) {
+        TEST_CALLOC(custom_key_data, PSA_BITS_TO_BYTES(bits) + 1);
```

Comment:
Non-blocker: why do +1 here instead of passing `PSA_BYTES_TO_BITS(MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE + 1)` as the `bits` argument?